example/src/test_mapping_data_loader.cc

```diff
@@ -0,0 +1,143 @@
+  m_contig_end_2_idx = vid_mapper.get_contig_end_offsets();
+  m_row_idx_to_info = vid_mapper.get_callsets();
+  m_callset_name_to_row_idx = vid_mapper.get_name_to_idx_map();
+}
+
+void MappingDataLoaderTester::validate_contig_info() {
+  std::cout <<"------------------------------------------------------\n";
+  int64_t prev_offset = -1;
+
+  for (std::vector<ContigInfo>::iterator it = m_contig_idx_to_info.begin(); it != m_contig_idx_to_info.end(); ++it) {
+    std::cout <<it->m_contig_idx <<" - " <<it->m_name <<" - " <<it->m_tiledb_column_offset <<" - " <<it->m_length <<"\n";
+    assert(it->m_tiledb_column_offset > prev_offset);
+    prev_offset = it->m_tiledb_column_offset;
+  }
+
+  std::cout <<"------------------------------------------------------\n";
+  int64_t prev_begin_first = -1;
+
+  for (std::vector<std::pair<int64_t, int>>::iterator it = m_contig_begin_2_idx.begin(); it != m_contig_begin_2_idx.end(); ++it) {
+    std::cout <<it->first <<" - " <<it->second <<"\n";
+    assert(it->first > prev_begin_first);
+    prev_begin_first = it->first;
+  }
+
+  std::cout <<"------------------------------------------------------\n";
+  int64_t prev_end_first = -1;
+
+  for (std::vector<std::pair<int64_t, int>>::iterator it = m_contig_end_2_idx.begin(); it != m_contig_end_2_idx.end(); ++it) {
+    std::cout <<it->first <<" - " <<it->second <<"\n";
+    assert(it->first > prev_end_first);
+    prev_end_first = it->first;
+  }
+
+  std::cout <<"------------------------------------------------------\n";
+  return;
+}
+
+void MappingDataLoaderTester::validate_callset_info() {
+  std::cout <<"------------------------------------------------------\n";
+  int64_t index = -1;
+
+  for (std::vector<CallSetInfo>::iterator it = m_row_idx_to_info.begin(); it != m_row_idx_to_info.end(); ++it) {
+    index++;
+    std::cout <<it->m_row_idx <<" - " <<it->m_name <<" - " <<it->m_file_idx <<" - " <<it->m_idx_in_file <<"\n";
+    assert(index == it->m_row_idx);
+    assert(m_callset_name_to_row_idx[it->m_name] == index);
+  }
+
+  std::cout <<"------------------------------------------------------\n";
+  return;
+}
+
+void get_input(const std::string param_name, const std::string param_value, std::string& location) {
+  std::string value;
+  std::cout <<"Enter value for <" <<param_name <<"> - type DEFAULT to use default value: ";
+  std::cin >> value;
+  if (0 == value.compare("DEFAULT")) {
+      location = param_value;
+  } else {
+      location = value;
+  }
+  return;
+}
+
+int main(int argc, char *argv[]) {
+  std::cout <<"------------ MappingData Loader - BEGIN -----------------\n";
+  SQLVidMapperRequest request;
+
+  get_input("host_name", "localhost", request.host_name);
+  get_input("user_name", "postgres", request.user_name);
+  get_input("pass_word", "postgres", request.pass_word);
+  get_input("db_name", "gendb", request.db_name);
+
+  get_input("work_space", "/home/rmantrix/git_repos/NomixDB/INST_GenomicsDB/tests/workspace", request.work_space);
```

Comment:
Hard-coded path :(
Even if this is default - no hard-coding for workspace, array, please. localhost for mapping DB is fine.